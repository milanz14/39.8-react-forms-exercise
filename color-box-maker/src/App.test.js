import React from "react";
import { render } from "@testing-library/react";
import App from "./App";

test("it should render without crashing", () => {
    render(<App />);
});

it("matches the snapshot", () => {
    const { asFragment } = render(<App />);
    expect(asFragment()).toMatchSnapshot();
});
