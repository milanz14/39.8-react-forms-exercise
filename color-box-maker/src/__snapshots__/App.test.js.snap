// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches the snapshot 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <div>
      <div>
        <form>
          <div>
            <label
              for="color"
            >
              Input Color
            </label>
            <input
              id="color"
              name="color"
              placeholder="Color"
              type="text"
              value=""
            />
          </div>
          <div>
            <label
              for="height"
            >
              Input Height
            </label>
            <input
              id="height"
              name="height"
              placeholder="Height"
              type="text"
              value=""
            />
          </div>
          <div>
            <label
              for="width"
            >
              Input Width
            </label>
            <input
              id="width"
              name="width"
              placeholder="Width"
              type="text"
              value=""
            />
          </div>
          <button>
            Make a new Box
          </button>
        </form>
      </div>
      <h2>
        Add Box Below
      </h2>
    </div>
  </div>
</DocumentFragment>
`;
